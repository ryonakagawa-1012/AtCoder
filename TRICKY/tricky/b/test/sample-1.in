3
1 -3 2
-10 30 -20
100 -300 200
