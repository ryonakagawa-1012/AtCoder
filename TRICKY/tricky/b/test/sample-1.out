2 1.000 2.000
2 1.000 2.000
2 1.000 2.000
